build.gradle

```diff
@@ -76,6 +73,12 @@ subprojects {
         jmhVersion = '1.19'
     }
 
+    compileJava {
+        sourceCompatibility = jdkVersion
+        targetCompatibility = jdkVersion
```

Comment:
Shouldn't this stay 1.8 in all cases?

Reply:
jdkVersion 1.9 used only for oraclejdk9 image on Travis
Why you want to use 1.8 in this case?

Reply:
I don't have any objections to compiling and running with JDK 9, that's a useful test.

But I don't think we want JDK 9 features used in source, or JDK9 specific jars.  Was this the aim?

Reply:
Got it, no it's just compatibility test.